develop-docs/application-architecture/dynamic-sampling/the-big-picture.mdx

```diff
@@ -6,29 +6,37 @@ sidebar_order: 1
 
 ![Sequencing](./images/sequencing.png)
 
+
+<Alert title="💡 Note" level="info">
+
+Dynamic Sampling currently operates on either spans or transactions, based on the feature flag `dynamic-sampling-spans`. The logic between the two event types is similar, so most of this documentation is kept at a generic level and important differences are pointed out explicitly. 
```

Comment:
The feature flag will be tied to what is the primary stats and billing metric according to the organization's plan. The default going forward is spans.